pygenometracks/tracks/BedGraphTrack.py

```diff
@@ -216,6 +221,7 @@ def plot(self, ax, chrom_region, start_region, end_region):
             score_list, x_values = self.get_values_as_bdg(score_list,
                                                           pos_list)
 
+        # I do not see how this part of the code can be executed:
```

Comment:
I think we should remove this part as I do not see how is it possible that self.properties['extra'] is an array...